packages/gallery/src/components/item/videos/videoItem.js

```diff
@@ -22,10 +30,19 @@ class VideoItem extends React.Component {
     };
   }
 
+  playingFromIframeClick = false;
```

Comment:
Why do you hold a property inside a component. It should be dependent on `state` and `props` only

Reply:
the reason I chose to store this variable outside the state is that it does not effect rendering, and only effect the logic of the youtube controller which is independent of the react rendering process